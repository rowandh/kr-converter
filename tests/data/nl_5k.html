
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-3-27737822</p></td>
                                <td height="26" ><p align="center">&nbsp;2024-12-27 오후 11:53:28</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>abkhgxmtc8 (10 원 페어)</b></p></td>
<!--								<td ><p align="center">&nbsp;85230</p></td>   -->
								<td ><p align="center">&nbsp;85,230</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>dsjgoqpeg</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>9 <font color='red'>◆</font>3 ♣3 ♠10 <font color='red'>♥</font>10 [10 3 투 페어]  (다이) (Big Blind)
<br>* 시작 : [StageNo:27737822] [Credit:462,970원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[dsjgoqpeg]
<br>* 앤티: -5,000원(457,970원)
<br>* 홀 카드딜: <font color='red'>♥</font>9(34) <font color='red'>◆</font>3(15) [9 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:9 탑(<font color='red'>♥</font>9 <font color='red'>◆</font>3)]
<br>* 베팅: [블라인드:BIG] [금액:5,000원] [Creadit:452,970원]
<br>* 베팅: 체크 -0원(452,970원) - 베팅순서: [0][9]
[1082ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>9<font color='red'>◆</font>3) C (♣3♠10<font color='red'>♥</font>10)
<br>* 턴 시작: [플랍(1)] [족보:10 3 투 페어(♠10 <font color='red'>♥</font>10 ♣3 <font color='red'>◆</font>3 <font color='red'>♥</font>9)]
<br>* 베팅: [베팅] (10,000원) Credit(442,970원) - 베팅순서: [1][2]
[6793ms]
<br>* 베팅: 다이 [1](442,970원) - 베팅순서: [1][7]
[7192ms]
<br>* 종료: WinMoney[0원] Credit[442,970원]
<br>* 결과: 패배 [족보:10 3 투 페어] [카드:♠10 <font color='red'>♥</font>10 ♣3 <font color='red'>◆</font>3 <font color='red'>♥</font>9] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-20,000</font></p></td>
											<td style="word-break:break-all"><p align="right">442,970</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>dks0001</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>9 <font color='red'>♥</font>3 [9 탑]  (다이) 
<br>* 시작 : [StageNo:27737822] [Credit:849,617원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:150]
<br>* NICKNAME:[dks0001]
<br>* 앤티: -5,000원(844,617원)
<br>* 홀 카드딜: <font color='red'>◆</font>9(21) <font color='red'>♥</font>3(28) [9 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:9 탑(<font color='red'>◆</font>9 <font color='red'>♥</font>3)]
<br>* 베팅: 다이 [0](844,617원) - 베팅순서: [0][1]
[2366ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>9<font color='red'>♥</font>3) C (♣3♠10<font color='red'>♥</font>10)
<br>* 종료: WinMoney[0원] Credit[844,617원]
<br>* 결과: 패배 [족보:9 탑] [카드:<font color='red'>◆</font>9 <font color='red'>♥</font>3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">844,617</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>okjvjbfh</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>8 ♠5 [8 탑]  (다이) 
<br>* 시작 : [StageNo:27737822] [Credit:2,710,987원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[ijcnvrioj]
<br>* 앤티: -5,000원(2,705,987원)
<br>* 홀 카드딜: <font color='red'>♥</font>8(33) ♠5(4) [8 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:8 탑(<font color='red'>♥</font>8 ♠5)]
<br>* 베팅: 다이 [0](2,705,987원) - 베팅순서: [0][3]
[298ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>8♠5) C (♣3♠10<font color='red'>♥</font>10)
<br>* 종료: WinMoney[0원] Credit[2,705,987원]
<br>* 결과: 패배 [족보:8 탑] [카드:<font color='red'>♥</font>8 ♠5] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">2,705,987</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>vkblrogib</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣K <font color='red'>◆</font>5 [K 탑]  (다이) 
<br>* 시작 : [StageNo:27737822] [Credit:614,375원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[vblnkjrgk]
<br>* 앤티: -5,000원(609,375원)
<br>* 홀 카드딜: ♣K(51) <font color='red'>◆</font>5(17) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(♣K <font color='red'>◆</font>5)]
<br>* 베팅: 다이 [0](609,375원) - 베팅순서: [0][2]
[3021ms]
<br>* 커뮤니티 카드 딜: H(♣K<font color='red'>◆</font>5) C (♣3♠10<font color='red'>♥</font>10)
<br>* 종료: WinMoney[0원] Credit[609,375원]
<br>* 결과: 패배 [족보:K 탑] [카드:♣K <font color='red'>◆</font>5] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">609,375</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>fycizoki</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠9 ♣6 ♣3 ♠10 <font color='red'>♥</font>10 [10 원 페어]  (다이) 
<br>* 시작 : [StageNo:27737822] [Credit:6,554,154원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[fycizoki]
<br>* 앤티: -5,000원(6,549,154원)
<br>* 홀 카드딜: ♠9(8) ♣6(44) [9 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:9 탑(♠9 ♣6)]
<br>* 베팅: 콜 -5,000원(6,544,154원) - 베팅순서: [0][6]
[2855ms]
<br>* 커뮤니티 카드 딜: H(♠9♣6) C (♣3♠10<font color='red'>♥</font>10)
<br>* 턴 시작: [플랍(1)] [족보:10 원 페어(♠10 <font color='red'>♥</font>10 ♠9 ♣6 ♣3)]
<br>* 베팅: 다이 [0](6,544,154원) - 베팅순서: [1][4]
[2844ms]
<br>* 종료: WinMoney[0원] Credit[6,544,154원]
<br>* 결과: 패배 [족보:10 원 페어] [카드:♠10 <font color='red'>♥</font>10 ♠9 ♣6 ♣3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">6,544,154</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>wertkhjljkwerth</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠6 <font color='red'>♥</font>4 [6 탑]  (다이) 
<br>* 시작 : [StageNo:27737822] [Credit:500,000원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[xryndke]
<br>* 앤티: -5,000원(495,000원)
<br>* 홀 카드딜: ♠6(5) <font color='red'>♥</font>4(29) [6 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:6 탑(♠6 <font color='red'>♥</font>4)]
<br>* 베팅: 다이 [0](495,000원) - 베팅순서: [0][4]
[207ms]
<br>* 커뮤니티 카드 딜: H(♠6<font color='red'>♥</font>4) C (♣3♠10<font color='red'>♥</font>10)
<br>* 종료: WinMoney[0원] Credit[495,000원]
<br>* 결과: 패배 [족보:6 탑] [카드:♠6 <font color='red'>♥</font>4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">495,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>abkhgxmtc8</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>Q ♠4 ♣3 ♠10 <font color='red'>♥</font>10 [10 원 페어]  (Entry Fee)
<br>* 시작 : [StageNo:27737822] [Credit:1,500,000원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[hfer23423]
<br>* 앤티: -5,000원(1,495,000원)
<br>* 홀 카드딜: <font color='red'>◆</font>Q(24) ♠4(3) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(<font color='red'>◆</font>Q ♠4)]
<br>* 신규 참가비 [금액:5,000원] [Credit:1,490,000원]
<br>* 베팅: 체크 -0원(1,490,000원) - 베팅순서: [0][5]
[5254ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>Q♠4) C (♣3♠10<font color='red'>♥</font>10)
<br>* 턴 시작: [플랍(1)] [족보:10 원 페어(♠10 <font color='red'>♥</font>10 <font color='red'>◆</font>Q ♠4 ♣3)]
<br>* 베팅: [하프] (55,000원) Credit(1,435,000원) - 베팅순서: [1][3]
[7478ms]
# 공베팅 반환 [45,000원]<br>* 종료: WinMoney[85,230원] Credit[1,565,230원]
<br>* 결과: 승리 [족보:10 원 페어] [카드:♠10 <font color='red'>♥</font>10 <font color='red'>◆</font>Q ♠4 ♣3] - 기권승

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>65,230</font></p></td>
											<td style="word-break:break-all"><p align="right">1,565,230</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>hbdjjjebv</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>7 <font color='red'>◆</font>4 ♣3 ♠10 <font color='red'>♥</font>10 [10 원 페어]  (다이) 
<br>* 시작 : [StageNo:27737822] [Credit:399,895원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:100]
<br>* NICKNAME:[hbdjjjebv]
<br>* 앤티: -5,000원(394,895원)
<br>* 홀 카드딜: <font color='red'>◆</font>7(19) <font color='red'>◆</font>4(16) [7 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:7 탑(<font color='red'>◆</font>7 <font color='red'>◆</font>4)]
<br>* 베팅: 콜 -5,000원(389,895원) - 베팅순서: [0][7]
[249ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7<font color='red'>◆</font>4) C (♣3♠10<font color='red'>♥</font>10)
<br>* 턴 시작: [플랍(1)] [족보:10 원 페어(♠10 <font color='red'>♥</font>10 <font color='red'>◆</font>7 <font color='red'>◆</font>4 ♣3)]
<br>* 베팅: 다이 [0](389,895원) - 베팅순서: [1][5]
[291ms]
<br>* 종료: WinMoney[0원] Credit[389,895원]
<br>* 결과: 패배 [족보:10 원 페어] [카드:♠10 <font color='red'>♥</font>10 <font color='red'>◆</font>7 <font color='red'>◆</font>4 ♣3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">389,895</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>mdsqqtuoo</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣5 <font color='red'>◆</font>2 ♣3 ♠10 <font color='red'>♥</font>10 [10 원 페어]  (다이) (Small Blind)
<br>* 시작 : [StageNo:27737822] [Credit:941,032원] [SB:5,000원] [BB:5,000원] [MBI:500,000원] [CBIR:300]
<br>* NICKNAME:[mdsqqtuoo]
<br>* 앤티: -5,000원(936,032원)
<br>* 홀 카드딜: ♣5(43) <font color='red'>◆</font>2(14) [5 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:5 탑(♣5 <font color='red'>◆</font>2)]
<br>* 베팅: [블라인드:SMALL] [금액:5,000원] [Creadit:931,032원]
<br>* 베팅: 체크 -0원(931,032원) - 베팅순서: [0][8]
[861ms]
<br>* 커뮤니티 카드 딜: H(♣5<font color='red'>◆</font>2) C (♣3♠10<font color='red'>♥</font>10)
<br>* 턴 시작: [플랍(1)] [족보:10 원 페어(♠10 <font color='red'>♥</font>10 ♣5 ♣3 <font color='red'>◆</font>2)]
<br>* 베팅: 체크 -0원(931,032원) - 베팅순서: [1][1]
[1282ms]
<br>* 베팅: 다이 [0](931,032원) - 베팅순서: [1][6]
[189ms]
<br>* 종료: WinMoney[0원] Credit[931,032원]
<br>* 결과: 패배 [족보:10 원 페어] [카드:♠10 <font color='red'>♥</font>10 ♣5 ♣3 <font color='red'>◆</font>2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">931,032</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-3-27737822">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


