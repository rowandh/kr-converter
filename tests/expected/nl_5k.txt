PokerStars Hand #15327737822:  Hold'em No Limit ($5000/$5000) - 2024/12/27 23:53:28 KST
Table 'Table 1' 9-max Seat #7 is the button
Seat 1: dks0001 ($849617 in chips)
Seat 2: vkblrogib ($614375 in chips)
Seat 3: okjvjbfh ($2710987 in chips)
Seat 4: wertkhjljkwerth ($500000 in chips)
Seat 5: abkhgxmtc8 ($1500000 in chips)
Seat 6: fycizoki ($6554154 in chips)
Seat 7: hbdjjjebv ($399895 in chips)
Seat 8: mdsqqtuoo ($941032 in chips)
Seat 9: dsjgoqpeg ($462970 in chips)
dks0001: posts the ante $5000
vkblrogib: posts the ante $5000
okjvjbfh: posts the ante $5000
wertkhjljkwerth: posts the ante $5000
abkhgxmtc8: posts the ante $5000
fycizoki: posts the ante $5000
hbdjjjebv: posts the ante $5000
mdsqqtuoo: posts the ante $5000
dsjgoqpeg: posts the ante $5000
mdsqqtuoo: posts small blind $5000
dsjgoqpeg: posts big blind $5000
abkhgxmtc8: posts big blind $5000
*** HOLE CARDS ***
dks0001: folds
vkblrogib: folds
okjvjbfh: folds
wertkhjljkwerth: folds
abkhgxmtc8: checks
fycizoki: calls $5000
hbdjjjebv: calls $5000
mdsqqtuoo: checks
dsjgoqpeg: checks
*** FLOP *** [3c Ts Th]
mdsqqtuoo: checks
dsjgoqpeg: bets $10000
abkhgxmtc8: raises $45000 to $55000
fycizoki: folds
hbdjjjebv: folds
mdsqqtuoo: folds
dsjgoqpeg: folds
Uncalled bet ($45000) returned to abkhgxmtc8
abkhgxmtc8 collected $85230 from pot
*** SUMMARY ***
Total pot $90000 | Rake $4770
Board [3c Ts Th]
Seat 1: dks0001 mucked [9d 3h]
Seat 2: vkblrogib mucked [Kc 5d]
Seat 3: okjvjbfh mucked [8h 5s]
Seat 4: wertkhjljkwerth mucked [6s 4h]
Seat 5: abkhgxmtc8 showed [Qd 4s] and won ($85230)
Seat 6: fycizoki mucked [9s 6c]
Seat 7: hbdjjjebv mucked [7d 4d]
Seat 8: mdsqqtuoo mucked [5c 2d]
Seat 9: dsjgoqpeg mucked [9h 3d]